# HG changeset patch
# User Rik <rik@octave.org>
# Date 1373737480 25200
#      Sat Jul 13 10:44:40 2013 -0700
# Node ID d429a05dc11ca21eaec0299c0dd81a0d65583cb8
# Parent  45afabc5f7c88ac424017967abd811b016ffd05e
legend.m: Move %!test block below %!demo blocks per standard usage.

* scripts/plot/legend.m: Move %!test block below %!demo blocks per standard
usage.

diff --git a/scripts/plot/legend.m b/scripts/plot/legend.m
--- a/scripts/plot/legend.m
+++ b/scripts/plot/legend.m
@@ -1175,30 +1175,16 @@ function updateline (h, d, hlegend, line
             "marker", marker, "markeredgecolor", get (h, "markeredgecolor"),
             "markerfacecolor", get (h, "markerfacecolor"),
             "markersize", get (h, "markersize"), "linestyle", "none",
             "userdata", h, "parent", hlegend);
     endif
   endif
 endfunction
 
-%!test
-%! graphics_toolkit ("gnuplot");
-%! h = figure ("visible", "off", "__graphics_toolkit__", "gnuplot");
-%! unwind_protect
-%!   position = get (h, "position");
-%!   plot (rand (3))
-%!   legend ()
-%!   filename = sprintf ("%s.eps", tmpnam ());
-%!   print (filename)
-%!   unlink (filename);
-%!   assert (get (h, "position"), position)
-%! unwind_protect_cleanup
-%!   close (h)
-%! end_unwind_protect
 
 %!demo
 %! clf;
 %! plot (rand (2));
 %! title ('legend called with cellstr and string inputs for labels');
 %! h = legend ({'foo'}, 'bar');
 %! legend location northeastoutside
 %! set (h, 'fontsize', 20);
@@ -1552,8 +1538,23 @@ endfunction
 %! subplot (3,1,3);
 %!  plot (rand (1,4));
 %!  set (gca (), 'xaxislocation', 'top');
 %!  xlabel ('xlabel');
 %!  ylabel ('ylabel');
 %!  legend ({'12345678901234567890'}, 'location', 'southwestoutside');
 %!  legend (option);
 
+%!test
+%! graphics_toolkit ("gnuplot");
+%! h = figure ("visible", "off", "__graphics_toolkit__", "gnuplot");
+%! unwind_protect
+%!   position = get (h, "position");
+%!   plot (rand (3));
+%!   legend ();
+%!   filename = sprintf ("%s.eps", tmpnam ());
+%!   print (filename);
+%!   unlink (filename);
+%!   assert (get (h, "position"), position);
+%! unwind_protect_cleanup
+%!   close (h);
+%! end_unwind_protect
+
